lib/std/fmt.zig

```diff
@@ -1223,7 +1224,14 @@ pub fn formatInt(
 
 pub fn formatIntBuf(out_buf: []u8, value: anytype, base: u8, case: Case, options: FormatOptions) usize {
     var fbs = std.io.fixedBufferStream(out_buf);
-    formatInt(value, base, case, options, fbs.writer()) catch unreachable;
+
+    formatInt(value, base, case, FormatOptions{
+        .alignment = options.alignment,
+        .fill = options.fill,
+        .precision = options.precision,
+        .width = options.width,
+        .signed = false,
```

Comment:
This disregards the caller's preference in `options.signed`.

Reply:
:facepalm: :+1: Thanks!